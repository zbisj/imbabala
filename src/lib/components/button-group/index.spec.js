import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import ButtonGroup from ".";

describe("ButtonGroup", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <ButtonGroup
          items={[
            {
              variant: "primary",
              label: "Primary",
              onClick: () => {
                console.log("Button Cliked");
              },
            },
            {
              variant: "success",
              label: "Success",
              onClick: () => {
                console.log("Button Cliked");
              },
            },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
