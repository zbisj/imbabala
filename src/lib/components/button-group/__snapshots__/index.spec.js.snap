// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders correctly 1`] = `
<div
  className=""
>
  <button
    className=""
    onClick={[Function]}
  >
    Primary
  </button>
  <button
    className=""
    onClick={[Function]}
  >
    Success
  </button>
</div>
`;
